lightning/src/routing/router.rs

```diff
@@ -2801,6 +2813,8 @@ where L::Target: Logger {
 					let exceeds_cltv_delta_limit = hop_total_cltv_delta > max_total_cltv_expiry_delta as u32;
 
 					let value_contribution_msat = cmp::min(available_value_contribution_msat, $next_hops_value_contribution);
+					// Verify the liquidity offered by this channel complies to the minimal contribution.
+					let contributes_sufficient_value = value_contribution_msat >= minimal_value_contribution_msat;
```

Comment:
I think this code move is unrelated? Can it be separated out into its own commit?